docs/CI-CD/ACR.md

```diff
@@ -0,0 +1,90 @@
+# Setup CI-CD with ACR
+
+> Setup Github Actions to build and push the helium container to Azure Container Registry (ACR)
+
+## Enable GitHub Actions
+
+By default, GitHub Actions are disabled for forked repositories. To enable:
+
+- Go to your forked repository on github.com
+- Click on the Actions tab
```

Comment:
Click on 'Settings' first, then Actions - but my newly forked repo was already set  for local and 3rd party actions for this repo.

Reply:
You don't need to click on Settings - Actions is a tab right next to Pull Requests